### PR TITLE
CheckOutlinesUFO pens work

### PR DESCRIPTION
**Pull request by cjchapman**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -154,6 +154,7 @@ def run(self):
 		  'mutatorMath>=2.1.0', #2.1.0
 		  'ufolib>=2.1.1',
 		  'ufonormalizer>=0.3.2',
+		  'fontPens>=0.1.0.dev0'
```
Comment (anthrotype) [inline setup.py]: only that `fontPens` is not on PyPI yet
https://pypi.python.org/pypi?%3Aaction=search&term=fontPens
so running setup.py install or pip install will fail

Comment (cjchapman) [reply setup.py]: May I help with getting `fontPens` on PyPI?

Comment (anthrotype) [reply setup.py]: thanks, I can do it.
tracking it here https://github.com/robofab-developers/fontPens/issues/7

Comment (cjchapman) [reply setup.py]: Awesome. Thanks! Also, after thinking about these changes further, I started to wonder about the change in behavior, which lead to this issue:
https://github.com/robofab-developers/fontPens/issues/8

If it's not expected behavior, I'd be happy to help by working on this issue.

**Pull request merged**